_posts/2021-05-29-massiani21a.md

```diff
@@ -1,16 +1,10 @@
 ---
 title: On exploration requirements for learning safety constraints
-abstract: Safety is particularly challenging for dynamical systems since it requires
-  constraint satisfaction along trajectory predictions. Equivalent control constraints
-  for feedback control can be computed in the form of maximal positive invariant sets,
-  but they are typically very computationally demanding and sensitive to model uncertainty,
-  even for offline analysis. Recent work has explored learning such constraints directly
-  from data, but currently either require extensive domain knowledge, or suffers from
-  data-inefficiency. We propose a model-free exploration strategy to learn these constraints
-  only where relevant for a nominal control policy, instead of for the whole state-action
-  space. Our exploration strategy results from an analysis of the minimum convergence
-  requirements, which it prioritizes, and is on-policy, which allows it to be directly
-  applied to existing learning schemes.
+abstract: Enforcing safety for dynamical systems is challenging, since it requires constraint satisfaction along trajectory predictions.
+Equivalent control constraints can be computed in the form of sets that enforce positive invariance, and can thus guarantee safety in feedback controllers without predictions.
+However, these constraints are cumbersome to compute from models, and it is not yet well established how to infer constraints from data.
+In this paper, we shed light on the key objects involved in learning control constraints from data in a model-free setting.
+In particular, we discuss the family of constraints that enforce safety in the context of a nominal control policy, and expose that these constraints do not need to be accurate everywhere. They only need to correctly exclude a subset of the state-actions that would cause failure, which we call the critical set.
```

Comment:
```suggestion
  In particular, we discuss the family of constraints that enforce safety in the context of a nominal control policy, and expose that these constraints do not need to be accurate everywhere. They only need to correctly exclude a subset of the state-actions that would cause failure, which we call the critical set.
```